test/unit/metrics/test_metricframe_comprehensive.py

```diff
@@ -54,9 +55,13 @@ def test_3m_2sf_2cf():
     cf_array = np.stack([cf_1_arr, cf_2_arr], axis=1)
 
     # Create the target object
+    start = time.time()
     target = metrics.MetricFrame(metrics=metric_dict, y_true=y_true, y_pred=y_pred,
                                  sensitive_features=sf_frame,
                                  control_features=cf_array)
+    end = time.time()
+    elapsed = end - start
+    print(f"MetricFrame evaluation took {elapsed} secs")
```

Comment:
why are we printing in a test?

Reply:
My mistake. This was leftover from when I was using this test for performance testing. Have removed